control-plane/tests/test_service_persistence.py

```
@@ -161,6 +161,64 @@ def transaction(self) -> Iterator[None]:
 class ControlPlaneServicePersistenceTests(unittest.TestCase):
+    def _build_phase19_in_scope_case(
+        self,
+        *,
+        store: object | None = None,
+    ) -> tuple[object, AegisOpsControlPlaneService, CaseRecord, str, datetime]:
+        if store is None:
+            store, _ = make_store()
+        service = AegisOpsControlPlaneService(
+            RuntimeConfig(
+                postgres_dsn="postgresql://control-plane.local/aegisops",
+                wazuh_ingest_shared_secret="reviewed-shared-secret",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "wazuh_ingest_shared_secret"

(S106)